docs/guide/issues-help.txt

```diff
@@ -1,3 +1,59 @@
 =============
 Issues & Help
-=============
\ No newline at end of file
+=============
+
+Our developer community is vibrant and highly engaged, with wide experience using Node.js with MongoDB.
```

Comment:
Not a mandatory change, but this would read better if it said:

"Our developer community is vibrant and highly engaged, with **extensive** experience using Node.js with MongoDB"

rather than

"Our developer community is vibrant and highly engaged, with **wide** experience using Node.js with MongoDB"

Reply:
Please ignore the above comment as it involves a readability change rather than a code/technical change